doc/content/verification/bibfile.bib

```diff
@@ -1,12 +1,12 @@
 @techreport{longhurst1992verification,
-  title={Verification and Validation of TMAP4},
+  title={{V}erification and {V}alidation of {TMAP4}},
   author={Longhurst, GR and Harms, SL and Marwil, ES and Miller, BG},
   year={1992},
   institution={EG and G Idaho, Inc., Idaho Falls, ID (United States)}
 }
 
 @techreport{ambrosek2008verification,
-  title={Verification and Validation of TMAP7},
+  title={{V}erification and {V}alidation of {TMAP7}},
   author={Ambrosek, James and Longhurst, GR},
   year={2008},
   number = "INEEL/EXT-04-01657",
```

Comment:
The GitHub review system will not let me make comment on Lines 12 through 15 inclusive for a commit suggestion here, but we should adjust this entry to that shown below:

```
number = {INEEL/EXT-04-01657},
month={December},
institution={Idaho National Engineering and Environmental Laboratory}
```

With the `publisher` entry being removed, as it is not used in a `techreport` BibTeX type.